test/parallel/test_tensorflow_process_sets.py

```diff
@@ -23,88 +23,86 @@
 
 from base_test_tensorflow import *
 
+from common import mpi_env_rank_and_size
+
 _IS_TF2 = LooseVersion(tf.__version__) >= LooseVersion('2.0.0')
 _is_mac = platform.system() == 'Darwin'
 
 
-# Set environment variable to enable adding/removing process sets after initializing Horovod.
-os.environ["HOROVOD_DYNAMIC_PROCESS_SETS"] = "1"
-
-
 class TensorFlowProcessSetsTests(BaseTensorFlowTests):
     """
-    Tests for ops in horovod.tensorflow that add/remove process sets after initialization.
+    Tests for ops in horovod.tensorflow using multiple process sets.
     """
     def __init__(self, *args, **kwargs):
         super(TensorFlowProcessSetsTests, self).__init__(*args, **kwargs)
 
+    @classmethod
+    def setUpClass(cls):
+        """Initializes Horovod with two process sets"""
+        _, mpi_size = mpi_env_rank_and_size()
```

Comment:
You can use `horovod.runner.common.util.env.get_env_rank_and_size`, which is agnostic to the specific MPI kind.

I think general use of `common.mpi_env_rank_and_size` can be replaced with that.

Reply:
Thanks, I've replaced that for the new static process set tests. As it stands, the existing `test_process_sets_static.py` also needs to know whether it's going to run with MPI or Gloo before it initializes Horovod, so I've left it as it is for now.

BTW `get_env_rank_and_size()` could be useful to expose as general user API. Everybody who wants to use static process sets needs that information from some source before they initialize Horovod.